google-cloud-tools-plugin/resources/config.properties

```diff
@@ -19,3 +19,5 @@ feature.appengine.flex=true
 
 # Should be used in any Notifications explicitly produced by the plugin.
 notifications.plugin.groupdisplayid=Google App Engine plugin
+
+cloudsdk.min.version=131.0.0
```

Comment:
I think it's Java best practices (as evidenced by the red sign at the end of the line GitHub displays).

It's just adding an enter and committing and pushing again, why is that an issue? :)